index.html

```diff
@@ -74,7 +74,7 @@ <h3>
                 kids. Wall is placed on <a href="https://www.google.com/maps/place/50°02'46.6%22N+22°00'55.4%22E">Siemieńskiego
                 14 street</a>. More details you can find on <a
                     href="http://wspinanie.rzeszow.pl/sciany-wspinaczkowe/centrum-wspinaczkowe-flywall/">wspinanie.rzeszow.pl</a>
-                or on <a href="https://www.facebook.com/flywall.rzeszow/">Facebook</a>.
+                or on <a href="http://www.flywall.pl/">flywall.pl</a>.
```

Comment:
Why don't you use HTTPS?. The https://www.flywall.pl/ url works correctly.

Reply:
@dariuszzbyrad thanks for catch! Fixed.